src/app/config/thunks.js

```diff
@@ -145,3 +148,95 @@ export const approveCollectionRequest = (id, redirect) => dispatch => {
             collectionDetailsErrorNotifications.updateCollection(error);
         });
 };
+
+export const getUsersRequest = () => dispatch => {
+    users
+        .getAll({ active: true })
+        .then(response => {
```

Comment:
what if the response fails ?

Reply:
As per Tims PR comment handling of this has been moved to the file `utilities/api-clients/user.js` it will call off to fire a notification (depending on the error) e.g. "An unexpected error has occurred whilst trying to get users. Please try again shortly, if the error persists please contact an administrator." Then finally in here it will log the error to console